tests/e2e/map/map_spatial.spec.js

```diff
@@ -31,764 +52,1767 @@ test.describe.skip('Map: Spatial interactions', () => {
+
+          // Checking that the right number of results are loaded ensures that the route
+          // was fulfilled correctly with the succesfull paramCheck
+          await expect(page.getByText('Showing 2 of 2 matching collections')).toBeVisible()
+
+          // Draws the spatial on the map
+          await expect(page).toHaveScreenshot('4326-polygon-spatial-drawn.png', {
+            clip: screenshotClip
+          })
+        })
+      })
 
-        // Select the point spatial type
-        await page.getByRole('link', { name: 'Draw a coordinate on the map to select a spatial extent' }).click()
+      test.describe('When drawing a new polygon from the map controls', () => {
+        test('renders correctly', async ({ page }) => {
+          await interceptUnauthenticatedCollections({
+            page,
+            body: commonBody,
+            headers: commonHeaders,
+            additionalRequests: [{
+              body: polygonBody,
+              headers: {
+                ...commonHeaders,
+                'cmr-hits': '2'
+              },
+              paramCheck: (parsedQuery) => parsedQuery?.polygon?.[0]
+                ?.match(/-28\.\d+,-9\.\d+,-28\.\d+,-44\.\d+,-10\.\d+,-44\.\d+,-28\.\d+,-9\.\d+/)
+            }]
+          })
+
+          await page.goto('/')
+
+          // Select the polygon spatial type
+          await page.getByRole('button', { name: 'Search by spatial polygon' }).click()
+
+          // Add the polygon to the map
+          await page.mouse.click(1000, 500)
+
+          await page.waitForTimeout(200)
+          await page.mouse.click(1100, 700)
+
+          await page.waitForTimeout(200)
+          await page.mouse.click(1000, 700)
+
+          await page.waitForTimeout(200)
+          await page.mouse.click(1000, 500)
+
+          // Updates the URL
+          // eslint-disable-next-line max-len
+          await expect(page).toHaveURL(/search\?polygon\[0\]=-28\.\d+%2C-9\.\d+%2C-28\.\d+%2C-44\.\d+%2C-10\.\d+%2C-44\.\d+%2C-28\.\d+%2C-9\.\d+/)
+
+          // Populates the spatial display field
+          await expect(
+            page.getByTestId('filter-stack__spatial')
+              .locator('.filter-stack-item__secondary-title')
+          ).toHaveText('Polygon')
+
+          await expect(page.getByTestId('spatial-display_polygon')).toHaveText('3 Points')
+
+          // Checking that the right number of results are loaded ensures that the route
+          // was fulfilled correctly with the succesfull paramCheck
+          await expect(page.getByText('Showing 2 of 2 matching collections')).toBeVisible()
+
+          // Draws the spatial on the map
+          await expect(page).toHaveScreenshot('4326-box-spatial-drawn.png', {
+            clip: screenshotClip
+          })
+        })
+      })
 
-        // Add the point to the map
-        await page.mouse.click(1000, 500)
+      test.describe.skip('When editing a polygon', () => {
```

Comment:
```suggestion
      test.describe('When editing a polygon', () => {
```

Reply:
We decided not to edit spatial right now, leaving these as skipped